### PR TITLE
Display the signal candle analyzed in telegram.

### LINKED ISSUE #6915: Display the candle analyzed in telegram
## Describe your environment
(if applicable)

  * Operating system: macos big sur 11.2.3
  * Python Version:3.9 (`python -V`)
  * CCXT version: 1.66.66 (`pip freeze | grep ccxt`)
  * Freqtrade Version: eee337c (`freqtrade -V` or `docker-compose run --rm freqtrade -V` for Freqtrade running in docker)

## Describe the enhancement

Show the actual candle analyzed in telegram.

Personally I found it to be a challenge to see which candle was used to buy and sell.  I use an enhanced charting software and this has helped me tremendously. 

*Explain the enhancement you would like*

Show the actual candle analyzed in telegram.

I have forked the repo and made the changes, I can further refine the display if that helps and make it configurable if you would like. I have a deep understanding of the codebase and I have a few other changes which at some stage I can share which may help others. 

https://github.com/Surfableio/freqtrade/commit/84638496bb3c8330baa57fb00d152731bf2ee4e5

Thank you again, you guys have done an amazing job!!!

### LINKED ISSUE #6962: Add dry/live indicator and average entry/exit price to telegram notifications
## Describe the enhancement

@froggleston and I were talking a bit and we noticed that there are two issues in the freqtrade discord that may hurt new users:
1) people fabricating results by deleting bad trades from their databases
2) people purporting dry runs as live runs

This false information may lead to two problems:
1) it may demoralize new/old users who are honest and trying their best
2) worse: it may lead new users to purchase bad strategies from the users who are falsifying results. Ultimately getting scammed and feeling resentment toward the FT community. 

In an attempt to clarify result sharing on discord, I think it would be nice to have additional information indicating if the results are live or dry, the slippage that occurred for that particular entry/exit, and the number of deleted trades in the DB. This will not completely solve the issue but it will help the discord community identify those who are falsifying information. 

Do y'all think this would help? Or is it a waste of space and will just clutter the telegram messages? I want to hear your thoughts.

### PR DESCRIPTION
**Pull request by FlockrushAdmin**
All the tests completed and test result files are attached. Thank you all!

## Summary

Show the actual candle analyzed in telegram.

Solve the issue: # 6915

Update freqtradebot.py to get and send the candle analyzed to create the entry or exit signal
Update telegram.py to send the candle analyzed to telegram

## What's new?

The PR displays the analyzed candle facilitating ease of discovery in the Binance portal the exact candle. This helps visually verify and provide ideas of improvement to the strategy. 

Screen Shot 2022-05-31 at 3 14 02 PM
[result_flake8.txt](https://github.com/freqtrade/freqtrade/files/8809024/result_flake8.txt)
[result_isort.txt](https://github.com/freqtrade/freqtrade/files/8809025/result_isort.txt)
[result_pytest.txt](https://github.com/freqtrade/freqtrade/files/8809026/result_pytest.txt)
[result_type-hints.txt](https://github.com/freqtrade/freqtrade/files/8809027/result_type-hints.txt)

### CODE REVIEW

**Comment by eSeR1805** [comment]
Maybe add a config toggle in the telegram section for people to choose if they want this extra reporting or not?
Feels like this is relevant for your usecases but for others(like me) it might be just extra clutter.

**Comment by FlockrushAdmin** [comment]
> Maybe add a config toggle in the telegram section for people to choose if they want this extra reporting or not? Feels like this is relevant for your usecases but for others(like me) it might be just extra clutter.

Will do. Thank you!!!

**Comment by xmatthias** [comment]
i'm ... not entirely sure how to approach this yet.

i don't really like the formatting (it's just a json blob ...) which especially on a phone, will look quite bad as it will wrap 'wherever' ... (but wait and read on)...

i can see it can provide benefits in some areas - but for other usecases, i fear it'll not go far enough.

People will want to see 'their' indicators, too - so maybe make that 'switch' a bit more dynamic - like 
'off' - default
'ohlcv' -> what you show now
'full' -> unfiltered candle

**Comment by FlockrushAdmin** [comment]
> 



> i'm ... not entirely sure how to approach this yet.
> 
> i don't really like the formatting (it's just a json blob ...) which especially on a phone, will look quite bad as it will wrap 'wherever' ... (but wait and read on)...
> 
> i can see it can provide benefits in some areas - but for other use cases, i fear it'll not go far enough.
> 
> People will want to see 'their' indicators, too - so maybe make that 'switch' a bit more dynamic - like 'off' - default 'ohlcv' -> what you show now 'full' -> unfiltered candle

Agreed better formatting and a more detailed raw option can be easier to digest and useful. 

I will add a more detailed version with the suggested test coverage. 

Have an awesome week!!!

File: freqtrade/templates/strategy_analysis_example.ipynb [hunk 1]
```diff
@@ -19,9 +19,27 @@
...
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 2,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "ename": "OperationalException",
+     "evalue": "Directory `/Users/surfer/Software/MMM/develop/freqtrade/freqtrade/templates/user_data` does not exist. Please use `freqtrade create-userdir` to create a user directory",
```
Comment (xmatthias) [inline freqtrade/templates/strategy_analysis_example.ipynb:27]: Please make sure to only commit what you'd like to have included.
a jupyter notebook that's been executed is clearly not among these.

if (intentional) changes to the notebook happened - please follow the [developer guideline](https://www.freqtrade.io/en/latest/developer/#updating-example-notebooks) around this.

File: .gitignore [hunk 2]
```diff
@@ -105,3 +105,8 @@ target/
 !config_examples/config_ftx.example.json
 !config_examples/config_full.example.json
 !config_examples/config_kraken.example.json
+doc/doc-filelist.js
+.gitignore
+.gitignore
+doc/doc-style.css
+doc/doc-script.js
```
Comment (xmatthias) [inline .gitignore:112]: I don't think these files make sense to include in a gitignore file.

they're not part of the repository - and excluding .gitignore is kinda dangerous.

**Review by xmatthias — CHANGES_REQUESTED** [review]
Please make sure to only include data you want - `git commit -a` should only be used once you've confirmed that the changes include what you intend to contribute.

**Comment by xmatthias** [comment]
As this branch saw no progress, and pushing to this branch is disabled (most likely implicitly, as the PR was done from your develop branch) - i've added the necessary changes and merged the changes directly.

It's now "optin" - so people have to explicitly enable this - as i think it'll clutter people's results without much reason (this is often non-necessary information).

**Pull request merged**